content/index.mdx

```diff
@@ -9,7 +9,7 @@ Primer's interface guidelines are a collection of principles, standards, and usa
 
 ## [Foundations](https://primer.style/design/foundations)
 
-The fundamental parts of the design system, such as color and typography, that underpin all GitHub interfaces.
+The fundamental parts of the design system that underpin all GitHub interfaces, such as color and typography.
```

Comment:
Not entirely sure why this is here when this is the most up to date thing for index.mdx and I didn't change it 😅

Reply:
It was a separate demo PR that I opened for a video.